prometheus/histogram.go

```diff
@@ -821,6 +830,11 @@ func (h *histogram) Write(out *dto.Metric) error {
 				Length: proto.Uint32(0),
 			}}
 		}
+
+		his.Exemplars = make([]*dto.Exemplar, 0)
```

Comment:
```suggestion
		his.Exemplars = make([]*dto.Exemplar, len(h.nativeExemplars.exemplars))
```
We could pre-allocate the array size here :)